src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -9,6 +9,7 @@
     <OutputPath>$(BinFolder)$(Configuration).FunctionalTests</OutputPath>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="..\..\..\ValueTypeConverterTests.cs" Link="ValueTypeConverterTests.cs" />
```

Comment:
Please keep the test class with other classes following same architecture.

Reply:
done